# fp: [0.0, 0.0, 0.0]
# fi: [0.0, 0.0, 0.0]
# fd: [0.0, 0.0, 0.0]

fp: [1.0, 1.0, 1.0]
fi: [0.008, 0.008, 0.008]
fd: [0.0, 0.0, 0.0]

ori_x: [8.0, 0.0, 0.0]
ori_y: [8.0, 0.0, 0.4]
ori_z: [3.0, 0.2, 0.0]

md: [0.0, 0.0, 0.0]
mp: [0.0, 0.0, 0.0]
mi: [0.0, 0.0, 0.0]

imp_d_: 0. #30. #7.5 #30.0 #31.6228
imp_k_: 0. #50.0 #12.5 #50.0 #50.
imp_scale_: 5.0

imp_m_: 0.0 #5.0 #3.0 #4.9 #3.0

imp_f_: 0.1



# step_count : [100, 4000, 100, 4000]
# ramp_slopes : [2., 0., -2., 0.]
# steps : [0.02, 0.1, 0.02, 0.1]
# joint_torque_ctrl_id : 0


